gradle-jdks/src/main/java/com/palantir/gradle/jdks/PalantirCaPlugin.java

```diff
@@ -173,6 +173,10 @@ static List<Certificate> parseCerts(byte[] multipleCertificateBytes) {
                     continue;
                 }
 
+                if (e.getMessage().contains("no more data allowed for version 1 certificate")) {
```

Comment:
we may want to check that `e.getMessage()` doesn't return null prior to the `.contains` invocation